sphinx/tutorial/cylc/furthertopics/inheritance.rst

```diff
@@ -0,0 +1,621 @@
+Inheritance
+===========
+
+We have seen in the :ref:`runtime tutorial <tutorial-cylc-families>` how
+tasks can be grouped into families.
+
+In this tutorial we will look at nested families, inheritance order and
+multiple inheritance.
+
+
+Inheritance Hierarchy
+---------------------
+
+Create a new suite by running the command::
+
+   rose tutorial inheritance-tutorial
+   cd ~/cylc-run/inheritance-tutorial
+
+You will now have a ``suite.rc`` file that defines two tasks each representing
+a different aircraft, the Airbus A380 jumbo jet and the Robson R44 helicopter:
+
+.. image:: https://upload.wikimedia.org/wikipedia/commons/0/09/A6-EDY_A380_Emirates_31_jan_2013_jfk_%288442269364%29_%28cropped%29.jpg
+   :width: 49%
+   :alt: A380
+
+.. image:: https://upload.wikimedia.org/wikipedia/commons/2/2f/Robinson-R44_1.jpg
+   :width: 49%
+   :alt: R44
+
+.. code-block:: cylc
+
+   [scheduling]
+       [[dependencies]]
+           graph = a380 & r44
+
+   [runtime]
+       [[VEHICLE]]
+           init-script = echo 'Boarding'
+           pre-script = echo 'Departing'
+           post-script = echo 'Arriving'
+
+       [[AIR_VEHICLE]]
+           inherit = VEHICLE
+           [[[meta]]]
+               description = A vehicle which can fly.
+       [[AIRPLANE]]
+           inherit = AIR_VEHICLE
+           [[[meta]]]
+               description = An air vehicle with fixed wings.
+           [[[environment]]]
+               CAN_TAKE_OFF_VERTICALLY = false
+       [[HELICOPTER]]
+           inherit = AIR_VEHICLE
+           [[[meta]]]
+               description = An air vehicle with rotors.
+           [[[environment]]]
+               CAN_TAKE_OFF_VERTICALLY = true
+
+       [[a380]]
+           inherit = AIRPLANE
+           [[[meta]]]
+               title = Airbus A380 Jumbo-Jet
```

Comment:
Ideally we can add full-stops to the end of all titles and descriptions (so just lines 62, 66 and 366 need editing to add one), as tends to be the format for later family meta-data entries. I'm being pedantic so just ignore me if you want to but you have already made amendments to this aim so might as well :)